cmd/client-keystone-auth/main.go

```diff
@@ -150,13 +150,15 @@ func main() {
 	var url string
 	var domain string
 	var user string
+	var project string
 	var password string
 	var options gophercloud.AuthOptions
 	var err error
 
 	pflag.StringVar(&url, "keystone-url", os.Getenv("OS_AUTH_URL"), "URL for the OpenStack Keystone API")
 	pflag.StringVar(&domain, "domain-name", os.Getenv("OS_DOMAIN_NAME"), "Keystone domain name")
 	pflag.StringVar(&user, "user-name", os.Getenv("OS_USERNAME"), "User name")
+	pflag.StringVar(&project, "project-name", os.Getenv("OS_PROJECT_NAME"), "Keystone project name")
```

Comment:
The environment variable OS_PROJECT_NAME is valid for the project name in OpenStack as http://git.openstack.org/cgit/openstack/python-openstackclient/tree/README.rst#n85 so LGTM